infra/base-images/base-runner/coverage

```diff
@@ -131,7 +129,7 @@ for fuzz_target in $FUZZ_TARGETS; do
   if [[ $FUZZING_LANGUAGE == "go" ]]; then
     # Continue if not a fuzz target.
     if [[ $FUZZING_ENGINE != "none" ]]; then
-      grep "go test -run" $fuzz_target > /dev/null 2>&1 || continue
+      grep "FUZZ_CORPUS_DIR" $fuzz_target > /dev/null 2>&1 || continue
```

Comment:
Did you mean $FUZZ_CORPUS_DIR

Reply:
I mean `FUZZ_CORPUS_DIR` which is a string used by the golang coverage fuzz target (it reads the environment variable) cf https://github.com/google/oss-fuzz/blob/master/infra/base-images/base-builder/ossfuzz_coverage_runner.go#L25

Reply:
Do you have a better idea to test that this is indeed a golang coverage fuzz target ?